packages/ui/src/ui/pages/navigation/content/MapNode/MapNodesTableIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Icon from '../../../../components/Icon/Icon';
+import {getIconNameForType} from '../../../../utils/navigation/path-editor';
+
+const TRASH_PATH = '//tmp/trash';
+
+export function isTrashNode(item: any) {
+    return item.path === TRASH_PATH;
+}
+
+export function isLinkToTrashNode(item: any) {
+    return item.targetPath === TRASH_PATH;
+}
+
+export function renderMapNodesTableIcon(item: any) {
```

Comment:
`any`!
Let's keep the method on its previous place. And add call of corresponding `UIFacotry`-method, like:
```ts
export function renderMapNodesTableIcon(item) {
    let icon = UIFactory.getNavigationMapNodeSettings()?.renderNodeIcon(item);
    if (icon) {
       // do nothing
    } else if ...
}
```